src/main/java/core/basesyntax/service/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CsvFileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<FruitTransaction> readTransactionsFromFile(String filePath) {
+        if (filePath == null) {
+            throw new RuntimeException("filePath cannot be null");
+        }
```

Comment:
remove this check

Reply:
Removed. But why shouldn't I do this check? Is it not possible to get null input?